liquibase-core/src/main/java/liquibase/diff/output/changelog/core/MissingTableChangeGenerator.java

```diff
@@ -148,7 +148,7 @@ public int getPriority(Class<? extends DatabaseObject> objectType, Database data
             }
             columnConfig.setType(typeString);
 
-            if (column.isAutoIncrement()) {
+            if (column.isAutoIncrement() && column.getDefaultValue() == null) {
```

Comment:
This could be another place where this change might be best constrained to only Postgres databases.